### ISSUE TITLE
Problem loading sourcemap in 6.6.8

### ISSUE THREAD

**Issue by atomiks** [aholachek/react-flip-toolkit#86]
When importing `Core` like this:

```js
import Core from 'react-flip-toolkit/core';
```

Sourcemaps can't be loaded as they point to `react-flip-toolkit.js` rather than `index.js`

```bash
Server running at http://localhost:1234 
⚠️  Could not load existing sourcemap of "../../node_modules/react-flip-toolkit/core/index.js".
⚠️  Could not load existing sourcemap of "../../node_modules/react-flip-toolkit/core/index.js".
✨  Built in 2.11s.
```

**Comment by aholachek** [comment]
Oh yikes, thanks for letting me know. I'm moving to a monorepo for v7 and have split `flip-toolkit` out into [its own package](https://www.npmjs.com/package/flip-toolkit) and the [sourcemaps seem ok there.](https://unpkg.com/flip-toolkit@6.6.8/dist/index.js)
But it I will fix the broken sourcemaps for core by tonight.

**Comment by aholachek** [comment]
This should be fixed in v6.6.9, thanks again for catching it.

**Comment by atomiks** [comment]
Yup all good
